CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
-- First version of JWT Extension.
+- Second version of JWT Support.
```

Comment:
This is a bit misleading, there's no first version (released) yet. Maybe call it iteration? Or just add all under first version?

Reply:
done